### DISCUSSION TITLE
DocsContainer in vue possible ?

### DISCUSSION THREAD

**Discussion by Christo44** [storybookjs/storybook#22097] [Help]
Ive been looking at the react documentation and scouring github for storybook examples:

I came across this adjustment in preview.js which allows one to insert custom content after a doc page.
This fails in vue, vite storybook. 

With a direct import error `error loading dynamically imported module`

Can someone help ?
```
import { DocsContainer } from '@storybook/addon-docs'

export const parameters = {
docs: {
    container: ({ children, context }) => (
      <DocsContainer>
        {children}
        <div class="sbdocs-footer">
          <p>© Footer info 2023</p>
        </div>
      </DocsContainer>
    )
  },
}
```

**Comment by Exotelis** [comment]
It's been more than a year, but it might still help you :) (Or me when I have to do it in another project)

```
{/* BaseContainer.mdx */}
import { DocsContainer } from "@storybook/blocks";

<DocsContainer {...props}>

{props.children}

FOOTER

</DocsContainer>
```

And then add this to you `preview.ts`:

```js
import BaseContainer from "./BaseContainer.mdx";

const preview: Preview = {
  tags: ["autodocs"],
  parameters: {
    docs: {
      container: BaseContainer,
    },
  },
};

export default preview;

```
